src/sql/src/plan/notice.rs

```diff
@@ -54,6 +55,13 @@ impl PlanNotice {
                 );
                 Some(details)
             }
+            PlanNotice::WebhookValidationDoesNotUseColumns => {
+                let details = "
+                The expression provided to validate a webhook request does not \
```

Comment:
This does not seem to be covered by a test:
```
 CREATE SOURCE webhook_bytes_with_validation FROM WEBHOOK
  BODY FORMAT BYTES
  VALIDATE USING ( TRUE );
```

Reply:
I changed this notice to an error, and added a test case